cloudformation/media-atom-maker.yaml

```diff
@@ -199,6 +199,17 @@ Resources:
             Action:
             - kinesis:Get*
             - kinesis:DescribeStream
+      - PolicyName: PlutoIntegrationS3Policy
```

Comment:
can this be `PlutoIntegrationS3GetPolicy` to be more explicit?

Reply:
sorry I don't understand?

Reply:
on re-reading I think i get it now :)